### PR TITLE
Dlp v2 ab

### PR DESCRIPTION
**Pull request by birdsean**

### CODE REVIEW

**Comment by birdsean** [comment]
https://user-images.githubusercontent.com/13811962/109031996-7d99f580-7682-11eb-8dd7-040675b5f9d2.mp4

**Comment by birdsean** [comment]
https://user-images.githubusercontent.com/13811962/109032810-411ac980-7683-11eb-9523-e7199ca8226a.mp4

**Comment by birdsean** [comment]
https://user-images.githubusercontent.com/13811962/109037218-b9838980-7687-11eb-922f-66d7e78091be.mp4

File: scripts/config/templates/nginx.conf.erb [hunk 1]
```diff
@@ -167,20 +172,30 @@ http {
...
+    # this proxies the request to the live render service
     if ($http_x_should_prerender) {
       proxy_pass $rendertron_base$neighbor_url;
       break;
     }
+    
+    # if it is a rendertron request, on the dlp v2 arm, and has a DLP-like URL, then redirect rendertron with query param
+    set $dlp_v2_test_condition "${http_x_neighbor_rendertron}---${is_dlp_v2}---${request_uri}";
+    if ($dlp_v2_test_condition ~ ^v[0-9]*---true---\/[A-Z][a-zA-Z\-]*--[A-Z][a-zA-Z\-]*$) {
+      return 302 "${request_uri}?ab=1";
```
Comment (birdsean) [inline scripts/config/templates/nginx.conf.erb:197]: Should I make this query param more descriptive?

Comment (haisleyb) [reply scripts/config/templates/nginx.conf.erb:197]: Possibly if we plan on doing similar ab tests with new prerendered pages in the future but who knows how far down the line that would be

**Review by haisleyb — APPROVED** [review]

File: scripts/config/templates/nginx.conf.erb (hunk 1 above)
Comment (birdsean) [reply scripts/config/templates/nginx.conf.erb:197]: K, I'll make it more descriptive just to be safe

**Pull request merged**